homework21/Product.java

```diff
@@ -0,0 +1,55 @@
+import java.time.LocalDateTime;
+
+public class Product {
+    private String typ;
+    private double price;
+    private boolean discount;
+    private LocalDateTime timestamp;
```

Comment:
name `timestamp` doesn't explain what is it. Rename to `createdAt`.

Reply:
+++